BoostRunClub/Views/Controllers/RunningPageViewController.swift

```diff
@@ -0,0 +1,280 @@
+//
+//  RunningViewController.swift
+//  BoostRunClub
+//
+//  Created by Imho Jang on 2020/11/23.
+//
+
+import AVFoundation
+import Combine
+import MapKit
+import UIKit
+
+final class RunningPageViewController: UIPageViewController {
+    enum Pages: CaseIterable {
+        case map, info, splits
+    }
+
+    private var pages = [UIViewController]()
+    private lazy var pageControl = makePageControl()
+    private lazy var backButton = makeBackButton()
+
+    private var viewModel: RunningPageViewModelTypes?
```

Comment:
```suggestion
    private let viewModel: RunningPageViewModelTypes
```

init 시점에 viewModel이 주입되기 떄문에 optional이 아닌 타입으로 사용 가능합니다.